applications/acdc/src/acdc_agent_fsm.erl

```diff
@@ -936,10 +936,10 @@ answered({'sync_req', JObj}, #state{agent_listener=AgentListener
     {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{member_call_id=CallId}=State) ->
     lager:info("caller channel ~s unbridged", [CallId]),
-    {'next_state', 'wrapup', State#state{wrapup_ref=wrapup_timer(State)}};
+    {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{agent_call_id=CallId}=State) ->
     lager:info("agent channel unbridged"),
-    {'next_state', 'wrapup', State#state{wrapup_ref=hangup_call(State)}};
+    {'next_state', 'answered', State};
```

Comment:
We've been running this code in production for a while and haven't seen a case where this change affects the behaviour of the agent. Do you know of one? As far as the PR, it is important because the bridge always happens to unbridge from the agent side, regardless of who hangs up (which means this feature doesn't work as expected)